pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -510,6 +575,68 @@ profiles:
 				},
 			},
 		},
+		{
+			name: "v1beta3 with non-default global percentageOfNodesToScore",
+			data: []byte(`
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+percentageOfNodesToScore: 10
+`),
+			wantProfiles: []config.KubeSchedulerProfile{
+				{
+					SchedulerName:            "default-scheduler",
+					PercentageOfNodesToScore: nil,
+					Plugins:                  defaults.PluginsV1beta3,
+					PluginConfig: []config.PluginConfig{
```

Comment:
defaults.PluginConfigV1beta3

Reply:
It's `PluginsV1beta3`  https://github.com/kubernetes/kubernetes/blob/9bedff1147837118e1892c262992ccef42ea4041/pkg/scheduler/apis/config/testing/defaults/defaults.go#L155

Reply:
I'm referring to this one https://github.com/kubernetes/kubernetes/blob/9bedff1147837118e1892c262992ccef42ea4041/pkg/scheduler/apis/config/testing/defaults/defaults.go#L277

Reply:
OK, replaced `PluginConfig: ... `   with `PluginConfig: defaults.PluginConfigsV1beta3`